docs/internals/kinematic-chain-config.md

```diff
@@ -0,0 +1,318 @@
+---
+title: "Configuring Complex Kinematic Chains"
+linkTitle: "Complex Kinematic Chains"
+weight: 70
+type: "docs"
+description: "How to write files defining kinematic parameters to configure intermediate reference frames for components with complex kinematic chains."
+tags: ["slam", "services"]
+# SMEs: Motion
+---
+
+Many components have complex kinematic chains and require an additional set of intermediate reference frames to use Viam's [motion service](/services/motion/).
+
+- For example, an [arm](/components/arm) has a reference frame originating where the arm is attached to a surface, but it also has links and joints whose frames of reference matter when attempting to move the arm to a [pose](/internals/pose) with [`MoveToPositions()`](/components/arm/#movetopositions).
+
+If you want to implement a component with a complex kinematic chain that is not already built into the RDK to build your robot with Viam, you need to add a file to your driver that details the attachment of the intermediate reference frames on the component.
```

Comment:
```suggestion
If you want to implement a component with a complex kinematic chain that is not already built into the RDK, you need to add a file to your driver that details the attachment of the intermediate reference frames on the component.
```

Hopefully, modular arms will be ready by the time people are doing this. At present, RDK modifications are needed to add an arbitrary component's physical description.

Reply:
👀 👍  done!